### PR TITLE
refactor(user): improve speed on list request

### LINKED ISSUE #102: Optimaliser is_TIHLDE_member
Uthenting av brukere går veldig tregt så vi må gjøre noe med is_TIHLDE_member ettersom den genereres basert på mye nesting i sqlbog dermed fører til dårlig performance.

### PR DESCRIPTION
**Pull request by olros**
## Proposed changes
- First removed the `is_TIHLDE_member` property in a try to increase speed/performance, but without luck
- The solution seems to be removing permissions from the serializer. I therefore created a new UserListSerializer without "permissions".
- Removed "app_token", "events", "badges", "groups", "unread_notifications" and "home_busstop" fields as well, couse I can't see why an admin user should need to see that info. Can discuss wether more fields also should be removed from the serializer.
- PR in Kvark will be created as well to support these changes

Issue number: closes #102 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ ] Build (`make PR`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/authentication/views.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def login(request):
     user = _try_to_get_user(user_id=user_id)
 
     if user.check_password(password):
-        if user.is_TIHLDE_member:
+        if user.memberships.filter(group__slug=Groups.TIHLDE).exists():
```
Comment (Eirsteir) [inline app/authentication/views.py:30]: Du kan bare beholde propertien siden det ikke var den som var problemet

**Review by Eirsteir — APPROVED** [review]
God analytisk arbeid 💪

File: app/authentication/views.py (hunk 1 above)
Comment (olros) [reply app/authentication/views.py:30]: Spørsmålet er om vi har bruk for den? Brukes ikke på frontend i det hele tatt

Comment (Eirsteir) [reply app/authentication/views.py:30]: Nei, men det er mye cleanere hvis den ligger på User som en property, slik som det var før. Du trenger ikke å legge den på serializeren

Comment (olros) [reply app/authentication/views.py]: Ahh, godt poeng! Legger tilbake 👍

**Pull request merged**